invenio_app_rdm/theme/assets/semantic-ui/js/invenio_app_rdm/landing_page/CommunitiesManagementDropdown.js

```diff
@@ -47,15 +80,35 @@ export class CommunitiesManagementDropdown extends Component {
       actionFeedback,
       pendingRequestModalOpen,
     } = this.state;
+
     const {
       userCommunitiesMemberships,
       searchConfig,
       recordCommunitySearchConfig,
       recordCommunityEndpoint,
-      toggleManageCommunitiesModal,
       recordUserCommunitySearchConfig,
     } = this.props;
 
+    const options = [
```

Comment:
I find this way of providing the options less readable, because you have to jump out of the JSX to check what options are provided to the dropdown in a separate variable. Could we keep the old way instead?